src/components/app/AppHeader.css

```diff
@@ -33,8 +33,13 @@
 
 .appHeaderGithubIcon {
   margin: 0 0.2em;
+  transition: opacity 250ms var(--animation-curve);
 }
 
 .appHeaderGithubIcon svg {
   vertical-align: middle;
 }
+
+.appHeaderGithubIcon:hover {
+  opacity: 0.5;
+}
```

Comment:
I realized, while looking at Markus' patch in #3407, that this icon had no visual indicator that it was clickable (besides the mouse cursor). So I added one :-)